src/System.Net.Requests/tests/HttpWebRequestTest.cs

```diff
@@ -25,13 +26,66 @@ namespace System.Net.Tests
 
     public partial class HttpWebRequestTest
     {
+        public class HttpWebRequestParameters
+        {
+            public DecompressionMethods AutomaticDecompression { get; set; }
+
+            public bool AllowAutoRedirect { get; set; }
+
+            public int MaximumAutomaticRedirections { get; set; }
+
+            public int MaximumResponseHeadersLength { get; set; }
+
+            public bool PreAuthenticate { get; set; }
+
+            public int Timeout { get; set; }
+
+            public SslProtocols SslProtocols { get; set; }
+
+            public bool CheckCertificateRevocationList { get; set; }
+
+            public void Configure(HttpWebRequest webRequest)
+            {
+                webRequest.AutomaticDecompression = AutomaticDecompression;
+                webRequest.AllowAutoRedirect = AllowAutoRedirect;
+                webRequest.MaximumAutomaticRedirections = MaximumAutomaticRedirections;
+                webRequest.MaximumResponseHeadersLength = MaximumResponseHeadersLength;
+                webRequest.PreAuthenticate = PreAuthenticate;
+                webRequest.Timeout = Timeout;
+            }
```

Comment:
Might be worth a comment that the other two properties aren't set because they're instead configured via a static

Reply:
All properties are now configured in one place.